prov/efa/docs/efa_rdm_protocol_v4.md

```diff
@@ -1445,6 +1449,22 @@ The binary format of a RUNTREAD_RTM packet's mandatory header is listed in table
 | `runt_length`    | 8 | integer | `uint64_t` | length of the application data |
 | `tag`            | 8 | integer | `uint64_t` | tag for RUNTREAD_TAGRTM only |
 
+### 4.6 RDMA-Write based data transfer (RDMA Write)
+
+The "RDMA-Write based data transfer" (RDMA write) is an extra feature that was
+introduced with the libfabric 1.18.0 release and was assigned the ID 5.  It is
+defined as an extra feature because there is a set of requirements (firmware,
+EFA kernel module and rdma-core) to be met before an endpoint can use the RDMA
+write capability, therefore an endpoint cannot assume the other party supports
+RDMA write.
+
+#### Direct write subprotocol
+
+The direct write subprotocol is the simplest write subprotocol in protocol v4.
+It does not involve a REQ packet.  The workflow is just for the writer to keep
+using RDMA write on the responder. For this protocol, it is not necessary for
+the responder to keep the progress engine running.
```

Comment:
> "It is not necessary for the responder to keep the progress engine running."

This is not true for write. `fi_write`   has a special case that requester can pass FI_REMOTE_CQ_DATA flag, which will cause the responder to get a CQ entry for the write. To support this  feature, the responder need to keep the progress engine running

Reply:
Is the updated comment accurate?